### ISSUE TITLE
Images being read before titles in search results previews

### ISSUE THREAD

**Issue by erinchambers** [CDRH/bandhistory_rails#46]
Deciding on alt text for our search result previews' thumbnail images has been difficult, in part because when we gave the images alt text, search results previews got read aloud by screen readers like this:

> link First page of Letter from Easterday to Lt. Dudley, March 22, 1886 Letter from Easterday to Lt. Dudley, March 22, 1886"

...and we definitely don't want that.

if we re-ordered the text and images in our search result previews so that the images come _after_ the rest of the content in the DOM. They can still look visually the same; the HTML order should change, however, and then we can use `order: 1` on the image and `order: 2` on the rest of the search result preview to get them to look the same as before.

We should fix this at the same time as #45.

**Comment by erinchambers** [comment]
I realized that we already had a solution for this in Family Letters (though not yet live on prod); the search results image should be `alt="" role="presentation"`, firstly because the item title is adjacent, and secondly because it makes the item title the first thing that's read aloud. I've updated the Rails view for the search results to reflect that solution.
